src/ui/src/components/core/layout/CoreTab.vue

```diff
@@ -180,12 +179,13 @@ const isTabActive = computed(() => {
 });
 
 onBeforeMount(() => {
-	if (isTabBit.value) return;
+	if (!isTabBit.value) return;
 	const tabContainerData = getTabContainerData();
-	const activeTab = tabContainerData.value?.activeTab;
-	if (activeTab) return;
-	if (!isComponentVisible(componentId, instancePath)) return;
-	tabContainerData.value = { activeTab: instancePath };
+	tabContainerData.value.tabs.push({
+		name: fields.name.value,
+		instancePath: getMatchingTabInstancePath(),
+		componentId: componentId,
+	});
```

Comment:
What is the reason here? 
Also I think use mutations not directly push.


```suggestion
	tabContainerData.value.tabs = [...tabContainerData.value.tabs, {
		name: fields.name.value,
		instancePath: getMatchingTabInstancePath(),
		componentId: componentId,
	}]
```

Reply:
> What is the reason here?

We use the list of tabs heavily in `CoreTabs.vue`

> Also I think use mutations not directly push.

We don't really need to spread the array because Vue can reliably detect array push operations ([reference](https://vuejs.org/guide/essentials/reactivity-fundamentals.html#deep-reactivity))